CHANGELOG.md

```diff
@@ -9,6 +9,27 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+- Added a new `Service`. This service will let you run OpenAI through 
+  OpenPipe's SDK.
+
+### Changed
+
+- `OpenPipe` can now be used. Can call OpenAI through OpenPipe SDK to get 
+  LLM logging stored in OpenPipe.
+
```

Comment:
I think this can just be in `Added` since nothing already existing has changed. I can take care of the change after merge.